tox.ini

```diff
@@ -20,13 +21,31 @@ commands =
 
 [testenv:type]
 skipsdist = true
-skip_install = true
+setenv =
+    DJANGO_CONFIGURATION = DevelopmentConfiguration
```

Comment:
If this is necessary, how about `TestingConfiguration`?

However, I still don't think it ought to be needed at all.

Reply:
Nope, not needed.